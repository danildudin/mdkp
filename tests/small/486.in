14 5 0
69 66 72 16 9 53 19 91 42 4 36 46 11 95 
81 66 46 37 37 8 29 21 49 31 87 73 14 10 
78 7 62 57 89 46 25 24 32 67 12 58 10 34 
39 54 27 68 96 37 48 4 39 89 92 88 32 2 
55 15 91 29 86 26 92 2 13 26 93 35 11 98 
15 19 18 18 1 19 26 20 21 9 12 6 22 5 
303 357 697 436 26 
