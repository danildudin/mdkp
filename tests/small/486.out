210
3
0 11 13 
